bitrise.yml

```diff
@@ -835,12 +835,7 @@ workflows:
             fastlane -version
 
             ./firefox-ios/l10n-screenshots.sh en-US
-        title: Generate screenshots
-    - deploy-to-bitrise-io@1.10:
-        inputs:
-        - deploy_path: l10n-screenshots-dd/
-        - is_compress: 'true'
-    - deploy-to-bitrise-io@1.10:
+    - deploy-to-bitrise-io@2.9.2:
```

Comment:
we need the l10n-screenshots-dd folder so that after L10nBuild the L10nScreenshots tests, which actually take the screenshots for each local run...